packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -92,10 +81,6 @@ describe('Withdraw ERC20 Token', () => {
         })
     })
 
-    it('should show ERC20 balance correctly', () => {
```

Comment:
After running the tests many times I found that this will fail on occasion. There's a chance for a tx to come through after we check the balance and it won't match.

We could always wait for the deposit to go through first but that could take time and make tests last longer.